### ISSUE TITLE
always disabled at startup

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [mlutfy/hidetopbar#175]
Hi,
I have a problem where every time I boot this extension is disabled and I have to manually turn it on in gnome-tweaks (or via the extension called "extensions"). Is there any logs or infos I can provide to help figure this out ?

Thanks and great job so far !

ubuntu : 18.04 LTS

**Comment by tuxor1337** [comment]
* Would you please provide your gnome-shell version?
* Do you use Wayland or X?
* Which other extensions do you use in parallel?
* Does the issue persist when all other extensions are disabled?
* Does the issue persist with a new user profile on the same system?

**Comment by thiswillbeyourgithub** [comment]
Sure !

* GNOME Shell 3.28.3
* I use X


* I use quite a lot of extensions : 
![screenshot from 2019-02-10 10-37-49](https://user-images.githubusercontent.com/26625900/52532139-e4f0e100-2d17-11e9-9799-cdcd60d33577.png)

* If you don't have any other idea I can try disabling all of them

* what do you call a user profile ?


Thanks for the quick reply ! (it won't be so fast on my end)

**Comment by tuxor1337** [comment]
Yes, you should disable all of them and if that solves you problem you should narrow down which of the other extension causes this. By user profile I mean a UNIX user account.

**Comment by thiswillbeyourgithub** [comment]
Okay so I disabled the top half of the extensions (all those above "hidetopbar" in the pic above) and rebooted and indeed hidetopbar was still on at boot. So I enabled them two by two two then rebooted to narrow down which were causing an issue.

I ended up with : all enabled before reboot -> reboot -> only "alternatetab","applicationmenu" and "dashtodock" that were disabled after the boot.

So I'll close this issue as I don't think it's related to your extension. If you have any idea what logs I peek into to have an idea why some extensions refuse to start that would be awesome :)

Have a nice day !
